packages/plugin-inline-script-content/inline-script-content.js

```diff
@@ -74,7 +74,7 @@ module.exports = {
       }
 
       // only attempt to grab some surrounding code if we have a line number
-      if (frame.lineNumber === undefined) return
+      if (!frame || frame.lineNumber === undefined) return
```

Comment:
oh, I did `if (frame === undefined || frame.lineNumber === undefined) return` for consistency but in case you are checking for `!frame`, might change the second condition to do the same? so...:
```suggestion
      if (!frame || !frame.lineNumber) return
```

Reply:
I left it as `=== undefined` because `0` is a valid number which evaluates as falsey. However, `0` isn't really a valid value for line number so arguably this is unnecessary.